index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About me</title>
+    <script src="https://kit.fontawesome.com/c98845bc10.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   <header class="headline-bar">
+    <div class="logo">
+        <h3>Zeeyah</h3>
+    </div>
+    <nav class="nav-links">
+        <ul>
+            <li><a href="#">Porfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+    </nav>
+    <div class="toogle">
+        <div class="menu-bar"></div>
+    </div>
+   </header>
+
+   <section class="intro">
+    <div class="intro-paragraph">
+        <h1>Hello there, I'm Fauziyah Bashiru, Welcome to my page</h1>
```

Comment:
- [ ] It is advisable to stick to the guiding template as much as possible. In order to achieve that in this regard, you must supply the exact equivalent info to this section, otherwise, it would not product the expected output. Kindly remove `Hello there`, and `welcome to my page` text and replace them with `I'm Fauziyah, Glad to see you!`